lib/Doctrine/Validator.php

```diff
@@ -108,6 +108,10 @@ public static function validateLength($value, $type, $maximumLength)
             $decimalPoint = $localeInfo['mon_decimal_point'] ? $localeInfo['mon_decimal_point'] : $localeInfo['decimal_point'];
             $e            = explode($decimalPoint, (string) $value);
```

Comment:
`explode` CAN return `false` if the delimiter is an empty string. The analyzers must have detected that this is possible in this case.